src/tests/ftest/vmd/led.py

```diff
@@ -0,0 +1,161 @@
+        except process.CmdError as details:
+            self.fail("dmg command failed: {}".format(details))
+        finally:
+            self.dmg.json.value = False
+
+        data = json.loads(result.stdout_text)
+        resp = data['response']
+        if data['error'] or len(resp['host_errors']) > 0:
+            if data['error']:
+                self.fail("dmg command failed: {}".format(data['error']))
+            else:
+                self.fail("dmg command failed: {}".format(resp['host_errors']))
+        uuid = []
+        for value in list(resp['host_storage_map'].values()):
+            if value['storage']['smd_info']['devices']:
+                for target in range(self.targets):
+                    uuid.append(value['storage']['smd_info']['devices'][target]['uuid'])
+        return uuid
+
+    def get_led_status_value(self, device_id=None):
+        """Get LED Status value.
+
+        Args:
+            device_id (str): Device UUID
+        Returns:
+            dmg LED status information
+        """
+        if device_id is None:
+            self.fail("No device id provided")
+
+        self.dmg.json.value = True
+        try:
+            result = self.dmg.storage_identify_vmd(uuid=device_id)
+        except process.CmdError as details:
+            self.fail("dmg command failed: {}".format(details))
+        finally:
+            self.dmg.json.value = False
+
+        data = json.loads(result.stdout_text)
+        resp = data['response']
+        if data['error'] or len(resp['host_errors']) > 0:
+            if data['error']:
+                self.fail("dmg command failed: {}".format(data['error']))
+            else:
+                self.fail("dmg command failed: {}".format(resp['host_errors']))
+        return resp
+
+    def set_device_faulty(self, device_id=None):
+        """Get a device to faulty state.
+
+        Args:
+            device_id (str): Device UUID
+        Returns:
+            dmg device faulty information.
+        """
+        if device_id is None:
+            self.fail("No device id provided")
+
+        self.dmg.json.value = True
+        try:
+            result = self.dmg.storage_set_faulty(uuid=device_id)
+        except process.CmdError as details:
+            self.fail("dmg command failed: {}".format(details))
+        finally:
+            self.dmg.json.value = False
+
+        data = json.loads(result.stdout_text)
+        resp = data['response']
+        if data['error'] or len(resp['host_errors']) > 0:
+            if data['error']:
+                self.fail("dmg command failed: {}".format(data['error']))
+            else:
+                self.fail("dmg command failed: {}".format(resp['host_errors']))
+        return resp
+
+    def test_vmd_led_status(self):
+        """Jira ID: DAOS-11290
+
+        :avocado: tags=all,daily_regression
+        :avocado: tags=hw,medium,ib2
+        :avocado: tags=vmd_led,faults
+        :avocado: tags=vmd_led_basic
```

Comment:
1. ib2 is no longer used
2. Will you please add a `vmd` feature tag for this new ftest directory?
3. Will you please use the test name as the unique tag?
4. Optionally, also add the class name?
```suggestion
        :avocado: tags=all,daily_regression
        :avocado: tags=hw,medium
        :avocado: tags=vmd,vmd_led,faults
        :avocado: tags=VmdLedStatus,test_vmd_led_status
```

Reply:
Done.